flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/SqlClient.java

```diff
@@ -85,16 +85,15 @@ private void start() {
             executor.start();
 
             // Open an new session
```

Comment:
nit: We can correct these obvious mistakes together： `an` -> `a`.  Btw, you can also pay attention to the warning of the idea checkStyle, like this class.